### PR TITLE
Basic FE scaffolding

### PR DESCRIPTION
**Pull request by archae0pteryx**
- Adds React and scaffolded FE code.
- Updated scripts
- Updated readme
[](https://media.giphy.com/media/58bVMncULfWf6B7SWy/giphy.gif)

### CODE REVIEW

File: artifacts/contracts/SchusterEtherFaucet.sol/SchusterEtherFaucet.dbg.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
   "_format": "hh-sol-dbg-1",
-  "buildInfo": "..\\..\\build-info\\55258aa257e10001bc97f022ae8cd24f.json"
+  "buildInfo": "../../build-info/447008838a9ef192abaaa97821684525.json"
```
Comment (Bschuster3434) [inline artifacts/contracts/SchusterEtherFaucet.sol/SchusterEtherFaucet.dbg.json:3]: Is this a difference between Linux and Window? should these .dbg files be .gitignored from the file?

Comment (archae0pteryx) [reply artifacts/contracts/SchusterEtherFaucet.sol/SchusterEtherFaucet.dbg.json:3]: hmm... i wonder how that will work. If you use the linux subsystem for windows i dont' think there'd be an issue

Comment (archae0pteryx) [reply artifacts/contracts/SchusterEtherFaucet.sol/SchusterEtherFaucet.dbg.json:3]: lets see what works? i don't think we should igone.

**Review by Bschuster3434 — APPROVED** [review]
Approved with one comment.

**Pull request merged**